config/environments/production.rb

```diff
@@ -94,7 +94,7 @@
 
   # Enable DNS rebinding protection and other `Host` header attacks.
   config.hosts = [
-    /content-publisher\..*gov.uk?/,
+    /content-publisher\..*\.gov.uk?/,
```

Comment:
```suggestion
    /content-publisher\..*\.gov.uk$/,
```
I'm not sure why I put the `?` originally, but a `$` is safer here as it'll restrict to only `.gov.uk` TLDs and not allow domains without the suffix.